RunningOrder/Shared/Components/FocusableTextField.swift

```diff
@@ -32,9 +32,12 @@ struct FocusableTextField: View {
         FocusableNSTextFieldRepresentable(placeholder: placeholder, value: $value, isFocused: $isFocused, onCommit: onCommit)
             .padding(.all, 5)
             // on focus background
-            .background(RoundedRectangle(cornerRadius: 10).foregroundColor(Color.white.opacity(isFocused ? 1 : 0)))
+            .background(RoundedRectangle(cornerRadius: 10)
+                            .foregroundColor(Color.white.opacity(isFocused ? 1 : 0)))
             // on focus border
-            .overlay(RoundedRectangle(cornerRadius: 10).strokeBorder(Color.accentColor, lineWidth: 1.0, antialiased: true).opacity(isFocused ? 1 : 0))
+            .overlay(RoundedRectangle(cornerRadius: 10)
+                        .strokeBorder(Color.accentColor, lineWidth: 1.0, antialiased: true)
+                        .opacity(isFocused ? 1 : 0))
```

Comment:
what about sharing `isFocused ? 1 : 0` between the two RoundedRectangles?

Reply:
which two ?

Reply:
I just re-indented.., but true, it could be better